src/app/core/services/label.service.ts

```diff
@@ -13,6 +13,11 @@ const COLOR_DARKNESS_THRESHOLD = 0.184;
 const COLOR_BLACK = '000000'; // Dark color for text with light background
 const COLOR_WHITE = 'ffffff'; // Light color for text with dark background
 
+export type simplifiedLabel = {
+  name: string;
+  color: string;
+};
```

Comment:
It might be more appropriate to move this to label.model instead of in label.service.